packages/flutter_tools/.analysis_options

```diff
@@ -0,0 +1,3 @@
+analyzer:
+  exclude:
+    - lib/loader.dart
```

Comment:
because this in the flutter_tools package the analyzer can't find the package:flutter stuff. We could make flutter_tools depend on package flutter or move this into package flutter.